### PR TITLE
docs: Some useful badges added

### PR DESCRIPTION
**Pull request by Fovty**
## Motivation

Badges to get an overview over some key values

## Changes

3 Badges added:

- current version
- monthly downloads
- test pipeline status

## TODO

- [x] I've assigned myself to this PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 new-pykube
 =========
 
+[![PyPI version](https://badge.fury.io/py/new-pykube.svg)](https://badge.fury.io/py/new-pykube) [![Downloads](https://static.pepy.tech/badge/new-pykube/month)](https://pepy.tech/project/new-pykube) ![Python Test](https://github.com/caas-team/new-pykube/actions/workflows/python-tests.yml/badge.svg)
```
Comment (jonathan-mayer) [inline README.md:4]: I would put each one in their own line. It shouldn't change anything, because Github ignores lines in most cases

Comment (jonathan-mayer) [inline README.md:4]: I would change the link to link to a respective site, not the badge site. 
PyPI version to the pipy site of new-pykube
Downloads to the GitHub releases
And Python Test to the last workflow results (log output)

**Review by jonathan-mayer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Fovty) [reply README.md]: Sounds good

Comment (Fovty) [reply README.md]: ok, updated the links

**Review by jonathan-mayer — APPROVED** [review]
LGTM

**Pull request merged**